mesonbuild/compilers.py

```diff
@@ -88,93 +88,93 @@ def is_library(fname):
-    gnulike_buildtype_linker_args.update({'plain' : [],
-                                          'debug' : [],
-                                          'debugoptimized' : [],
-                                          'release' : ['-Wl,-O1'],
-                                          'minsize' : [],
+    gnulike_buildtype_linker_args.update({'plain': [],
+                                          'debug': [],
+                                          'debugoptimized': [],
+                                          'release': ['-Wl,-O1'],
+                                          'minsize': [],
                                          })
 
-msvc_buildtype_linker_args = {'plain' : [],
-                              'debug' : [],
-                              'debugoptimized' : [],
-                              'release' : [],
-                              'minsize' : ['/INCREMENTAL:NO'],
+msvc_buildtype_linker_args = {'plain': [],
+                              'debug': [],
+                              'debugoptimized': [],
+                              'release': [],
+                              'minsize': ['/INCREMENTAL:NO'],
                               }
 
-java_buildtype_args = {'plain' : [],
-                       'debug' : ['-g'],
-                       'debugoptimized' : ['-g'],
-                       'release' : [],
-                       'minsize' : [],
+java_buildtype_args = {'plain': [],
+                       'debug': ['-g'],
+                       'debugoptimized': ['-g'],
+                       'release': [],
+                       'minsize': [],
                        }
 
-rust_buildtype_args = {'plain' : [],
-                       'debug' : ['-g'],
-                       'debugoptimized' : ['-g', '--opt-level', '2'],
-                       'release' : ['--opt-level', '3'],
-                       'minsize' : [],
+rust_buildtype_args = {'plain': [],
+                       'debug': ['-g'],
+                       'debugoptimized': ['-g', '--opt-level', '2'],
+                       'release': ['--opt-level', '3'],
+                       'minsize': [],
                        }
 
-d_gdc_buildtype_args = {'plain' : [],
-                       'debug' : ['-g', '-O0'],
-                       'debugoptimized' : ['-g', '-O'],
-                       'release' : ['-O3', '-frelease'],
-                       'minsize' : [],
+d_gdc_buildtype_args = {'plain': [],
+                       'debug': ['-g', '-O0'],
+                       'debugoptimized': ['-g', '-O'],
+                       'release': ['-O3', '-frelease'],
+                       'minsize': [],
                        }
 
-d_ldc_buildtype_args = {'plain' : [],
-                       'debug' : ['-g', '-O0'],
-                       'debugoptimized' : ['-g', '-O'],
-                       'release' : ['-O3', '-release'],
-                       'minsize' : [],
+d_ldc_buildtype_args = {'plain': [],
+                       'debug': ['-g', '-O0'],
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->